setup.py

```diff
@@ -1,72 +1,77 @@
 # -*- coding: utf-8 -*-
+"""Installer for the collective.glossary package."""
+
 from setuptools import find_packages
 from setuptools import setup
 
-version = '1.0b2.dev0'
-description = 'A Dexterity-based content type to define a glossary and its terms.'
-long_description = (
-    open('README.rst').read() + '\n' +
-    open('CONTRIBUTORS.rst').read() + '\n' +
-    open('CHANGES.rst').read()
+
+long_description = "\n\n".join(
+    [
+        open("README.rst").read(),
+        open("CONTRIBUTORS.rst").read(),
+        open("CHANGES.rst").read(),
+    ]
 )
 
+
 setup(
-    name='collective.glossary',
-    version=version,
-    description=description,
+    name="collective.glossary",
+    version="1.0a1",
+    description="A Dexterity-based content type to define a glossary and its terms.",
     long_description=long_description,
+    # Get more from https://pypi.org/classifiers/
     classifiers=[
-        'Development Status :: 4 - Beta',
-        'Environment :: Web Environment',
-        'Framework :: Plone',
-        'Framework :: Plone :: 4.3',
-        'Framework :: Plone :: 5.1',
-        'Intended Audience :: Education',
-        'Intended Audience :: End Users/Desktop',
-        'License :: OSI Approved :: GNU General Public License v2 (GPLv2)',
-        'Operating System :: OS Independent',
-        'Programming Language :: Python',
-        'Programming Language :: Python :: 2.7',
-        'Topic :: Software Development :: Libraries :: Python Modules',
+        "Environment :: Web Environment",
+        "Framework :: Plone",
+        "Framework :: Plone :: Addon",
+        "Framework :: Plone :: 5.2",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
```

Comment:
is it possible to keep Python 3.6 compatibility?

a good reference is what plone.restapi supports https://github.com/plone/plone.restapi/blob/master/setup.py#L62

but we need to add this in our CI matrix to make sure it still works

Reply:
yes. added.